src/main/java/dev/mja00/villagerLobotomizer/LobotomizeStorage.java

```diff
@@ -306,6 +306,10 @@ private void refreshTrades(@NotNull Villager villager) {
 
             if (this.restockSound != null) {
                 villager.getWorld().playSound(villager.getLocation(), this.restockSound, SoundCategory.NEUTRAL, 1.0F, 1.0F);
+            } else {
+                villager.getWorld().playSound(villager.getLocation(), VillagerUtils.PROFESSION_TO_SOUND.get(villager.getProfession()),
```

Comment:
`VillagerUtils.PROFESSION_TO_SOUND.get` can return `null` here due to being a nitwit, so we should handle if their profession is null.